content/docs/reference/x-forwarded-for-settings.mdx

```diff
@@ -0,0 +1,166 @@
+---
+id: x-forwarded-for-settings
+title: X-Forwarded-For Settings
+description: This page documents X-Forwarded-HTTP header settings.
+keywords:
+  - reference
+  - X-Forwarded-For settings
+  - x-forwarded-for-http-header
+  - x forwarded for http header
+  - the number of trusted hops
+  - xff_num_trusted_hops
+  - xff num trusted hops
+pagination_prev: null
+pagination_next: null
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# X-Forwarded-For Settings
+
+## X-Forwarded-For HTTP Headers
```

Comment:
I find this name pretty confusing. I'd suggest we standardize on "Skip XFF Append" for this (to match the config file key), but let me know what you think.

(I've also filed https://github.com/pomerium/internal/issues/1849 to reconsider how we label this setting in the Enterprise Console and in the Zero UI.)

Reply:
@kenjenkins I agree with you 100% on changing the setting name to match with the config file key.